anope_api/views/api/v0/auth.py

```diff
@@ -74,13 +98,17 @@ def get_request_data():
     if not request_data:
         raise NoData()
 
-    return request_data
+    return {
+        k: request_data[k] for k in request_data
```

Comment:
Why add a newline here? It messes with the ability to read the entire comprehension cohesively

Reply:
Style choice, I split conditions to a new line in complex comprehensions